src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -31,4 +30,11 @@ public String getSalaryInfo(String[] names, String[] data, String dateFrom, Stri
         }
         return builder.toString();
     }
+
+    private LocalDate parseDate (String date) {
+        LocalDate dateResult = LocalDate.of(Integer.parseInt(date.split("[.]")[2]),
+                Integer.parseInt(date.split("[.]")[1]),Integer.parseInt(date
+                        .split("[.]")[0]));
```

Comment:
Constants?)

Reply:
(Optional) It is okay for now, but in the future, it is better to move such low-level logic to a separate service class which will be responsible only for parsing